manifests/metac.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: metac.openebs.io/v1alpha1
+kind: GenericController
+metadata:
+  name: cspauto-controller
+spec:
+  watch:
+    apiVersion: dao.mayadata.io/v1alpha1
```

Comment:
Mention of `dao` somewhere will be helpful. What does it mean? Data Agility Operator ?? 
Is cStor auto controller and dao related or is the same thing ?